### ISSUE TITLE
Discussion: More accessibility features needed in samples

### ISSUE THREAD

**Issue by ArchieCoder** [microsoft/microsoft-ui-xaml#1164]
# Discussion: More accessibility features needed in samples
Windows Template Studio generated code and Microsoft UWP examples should demonstrate how accessibility should be done.

Someone pointed me to this good example https://github.com/microsoft/calculator/, but in long term, samples by Microsoft could include more accessibility work by default.

**Comment by mrlacey** [comment]
@ArchieCoder is there anything specific in the WinTS generated code that you are aware of that needs addressing? Or is this more of a general request for improvements?

**Comment by ArchieCoder** [comment]
Just general request. Adding more AccessibilityProperties.Name and maybe some Access Key, Accelleration Key so people can know how to add them in their app.

**Comment by SavoySchuler** [comment]
Here's a [link](https://twitter.com/archiecoder/status/1161083175826857984) to the original thread which sparked this discussion. Thanks for starting it here too, @ArchieCoder!

**Comment by YuliKl** [comment]
@ArchieCoder, are there specific Microsoft UWP examples you've come across that are failing to demonstrate accessibility best practices? It's easier to fix these problems if we know exactly where they are :)

Adding @predavid for visibility about the WTS comment.

**Comment by ArchieCoder** [comment]
@YuliKl 

1- Visit this section along all the sub-sections, I would say more samples please. https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/accessibility-overview

2- From the link above, there are 2 samples, but they were last updated in 2013! I can't even open them in the latest Visual Studio.

https://code.msdn.microsoft.com/windowsapps/XAML-accessibility-sample-d63e820d
https://code.msdn.microsoft.com/windowsapps/XAML-high-contrast-style-c0ce936f

**Comment by mrlacey** [comment]
While this was originally opened with reference to UWP, is it worth revisting in terms of WinUI3 support in TemplateStudio and with a review of current docs (& samples) coverage with regard to accessibility

**Comment by mrlacey** [comment]
pinging to keep open.
Accessibility is important, but developers don't know how to do it well. More samples, please.
